galette/templates/default/pages/member_form.html.twig

```diff
@@ -120,6 +120,24 @@
         </div>
         <div class="sixteen wide tablet ten wide computer eleven wide widescreen column">
     {% endif %}
+
+    {% if (member.id is empty or self_adh and members.list is not defined) and preferences.pref_bool_empty_form_link %}
```

Comment:
Not sure this one is useful; link from public pages should be enough.

Reply:
When self-subscription is disabled, I chose to replace the subscription link with the link to download the pdf :
![pdf-in-menu](https://github.com/user-attachments/assets/16763698-82c5-4f2c-ab6b-7d18d5cd0cce)

But when self-subscription is enabled, in my opinion, it is logically not necessary to provide the pdf link. Unless admins want to give the user the choice of subscribing by mail or in person (for those who are reluctant to online forms). That's why in that case I chose not to add a supplementary link in the menu, but rather a message above the registration form :
![pdf-in-message](https://github.com/user-attachments/assets/de2b887d-56f3-49af-a6ae-ddf85fadd393)

Reply:
OK, I did not understood correctly the goal; thanks for the explanation. Seems OK then ;)